### PR TITLE
CSS fixes and information message about the fork

### PR DESCRIPTION
**Pull request by obuisard**

### CODE REVIEW

File: administrator/components/com_j2store/helpers/help.php [hunk 1]
```diff
@@ -28,17 +28,22 @@ public static function getInstance(array $config = array())
...
+    $html = '<div class="video-tutorial panel panel-solid-info">
 				<p class="panel-body">'.JText::_('J2STORE_VIDEO_TUTORIALS_HELP_TEXT').'
 						 				<a class="btn btn-success" target="_blank" href="'.$video_url.'">
 						 					'.JText::_('J2STORE_WATCH').'</a>
 						 			</p>
 						 		</div>';
-		return $html;
-	}
+    $newhtml = '<div class="alert alert-info d-flex align-items-center" role="alert">
+            <i class="fas fa-solid fa-info-circle flex-shrink-0 me-2"></i>
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:41]:
```suggestion
            <span class="fas fa-solid fa-info-circle flex-shrink-0 me-2"></span>
```

File: administrator/components/com_j2store/helpers/help.php [hunk 2]
```diff
@@ -28,17 +28,22 @@ public static function getInstance(array $config = array())
...
+            <div>'.JText::_('J2STORE_TAKEOVER_INFO').'</div>
+            <a href="https://www.j2commerce.com" class="btn btn-sm btn-dark text-light  ms-3 ms-lg-auto" title="Visit J2Commerce" target="_blank">'.JText::_('J2STORE_FIND_OUT_MORE').'</a>
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:43]:
```suggestion
            <a href="https://www.j2commerce.com" class="btn btn-sm btn-dark text-light ms-3 ms-lg-auto" title="Visit J2Commerce" target="_blank"  rel="noreferrer noopener">'.JText::_('J2STORE_FIND_OUT_MORE').'</a>
```

File: media/j2store/css/J4/j2store_admin.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
-body.com_j2store .page-title {background:url(../images/j2store-icon.png) no-repeat 0 50%;padding-left:50px;}
-@media (max-width: 767px) { body.com_j2store .page-title {background:url(../images/j2store-icon.png) no-repeat center center;padding-left:0; text-indent: -800px} }
+body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat 0 50%;padding-left: 25px;margin-left: 25px;}
```
Comment (brianteeman) [inline media/j2store/css/J4/j2store_admin.css:1]:
```suggestion
body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat 0 50%;padding-inline-start: 25px;margin-inline-start: 25px;}
```

logical css operators ensire its both LTR and RTL compatible

File: media/j2store/css/J4/j2store_admin.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+@media (max-width: 767px) { body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat center center;padding-left:0; text-indent: -800px} }
```
Comment (brianteeman) [inline media/j2store/css/J4/j2store_admin.css:2]:
```suggestion
@media (max-width: 767px) { body.com_j2store .page-title {background:url(../../images/j2store-icon.png) no-repeat center center;padding-inline-start:0; text-indent: -800px} }
```

File: modules/admin/j2store_menu/tmpl/default.php [hunk 5]
```diff
@@ -92,16 +92,16 @@
...
-						'taxprofiles' => 'fa fa-sitemap',
-						'lengths' => 'fa fa-arrows-v',
-						'weights' => 'fa fa-arrows-h',
-						'orderstatuses' => 'fa fa-check-square'
+            'geozones' => 'fa fa-pie-chart fa-chart-pie',
+            'taxrates' => 'fa fa-calculator',
+            'taxprofiles' => 'fa fa-sitemap',
+            'lengths' => 'fa fa-arrows-alt-v fa-up-down',
+            'weights' => 'fa fa-arrows-alt-h fa-left-right',
+            'orderstatuses' => 'fa fa-check-square'
```
Comment (brianteeman) [inline modules/admin/j2store_menu/tmpl/default.php:104]:
```suggestion
						'geozones' => 'fa fa-pie-chart fa-chart-pie',
						'taxrates' => 'fa fa-calculator',
						'taxprofiles' => 'fa fa-sitemap',
						'lengths' => 'fa fa-arrows-alt-v fa-up-down',
						'weights' => 'fa fa-arrows-alt-h fa-left-right',
						'orderstatuses' => 'fa fa-check-square'
```

File: administrator/components/com_j2store/helpers/help.php (hunk 1 above)
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:41]:
```suggestion
            <span class="fas fa-solid fa-info-circle flex-shrink-0 me-2" aria-hidden="true"></span>
```

File: administrator/components/com_j2store/helpers/help.php (hunk 2 above)
Comment (j2commerce) [reply administrator/components/com_j2store/helpers/help.php]: Thanks Brian

**Pull request merged**